addon/components/pix-checkbox.js

```diff
@@ -22,7 +23,15 @@ export default class PixCheckbox extends Component {
   }
 
   get isDisabled() {
-    return this.args.isDisabled || this.args.disabled;
+    warn(
+      'PixCheckbox: @isDisabled attribute should be a boolean.',
+      [true, false, undefined, null].includes(this.args.isDisabled),
```

Comment:
il ne manque pas un `not` ? le fameux point d'exclamation  `!` ?

Reply:
J'ai ajouté des tests en stubbant `warn` pour s'assurer du résultat. On pourrait extraire ça ailleurs j'imagine par la suite

Reply:
je me suis posé la question dans un second temps 🤣